### PR TITLE
[WIP] Improve documentation

### PR DESCRIPTION
**Pull request by vvmruder**

### CODE REVIEW

**Comment by vvmruder** [comment]
@jwkaltz, @ger-benjamin Would it be possible for you to add a little doc about the print config? Especially about the part with buffer, DPI and the connected config.yml in the mapfish print part. I think this is a little under documented in the moment :)

File: doc/source/changes.rst [hunk 1]
```diff
@@ -0,0 +1,555 @@
...
+
+Version 1.1.0
+-------------
+
+The stable version 1.1.0 contains a lot of changes. It can be counted as the first version to be used in
+production mode. When you are updating from previous version to 1.1.0 you will have to adjust your yml file.
+Description below will try to classify new options whether they are *optional* or **mandatory** to use the
+new version.
+Of course you also could use the way described in the ``installation-step-configuration``. But then it will
+create a completely new yml but valid file. In order to that its up to your decision: Migrate new options to
```
Comment (ger-benjamin) [inline doc/source/changes.rst]: In order to *do* that ?

File: doc/source/changes.rst [hunk 2]
```diff
@@ -0,0 +1,555 @@
...
+OEREBlex
+^^^^^^^^
+
+This version includes binding/adapter to oereb lex. The code can be found
+`here <https://github.com/camptocamp/pyramid_oereb/blob/v1.1.0/pyramid_oereb/contrib/sources/plr_oereblex.py>`__.
+This should enable you to use OEREBlex with a minimum of configuration.
+See :ref:`changes-new-config-oereb-lex` for further details of configuration options.
+
+The idea of this oereb lex adapter is to access OEREBlex documents directly via API provided by OEREBlex.
+This you can configure this per theme. If you have a theme which has all documents stored in OEREBlex you
```
Comment (ger-benjamin) [inline doc/source/changes.rst]: ~~This~~ You ... ?

File: doc/source/changes.rst [hunk 3]
```diff
@@ -0,0 +1,555 @@
...
+.. note:: OREBlex uses unique ids for the document sets. To prevent from storing redundant data and for
+    simplifying things we decided to not store the complete link in database attribute but the id only!
+
+Regarding to the note above we assume you have stored the correct id of your OEREBlex document set per public
+law restriction in your database table.
+
+The next step is to define the sqlalchemy mappings. This works like normal mapping definition described here:
+:ref:`configuration-adapt-models`. The difference to the standard database configuration is here:
+
+- all document related model classes are obsolete
```
Comment (ger-benjamin) [inline doc/source/changes.rst]: Missing final dot.
Then missing major letter to start both lines under

File: doc/source/changes.rst [hunk 4]
```diff
@@ -0,0 +1,555 @@
...
+- the public law restriction class gets a attribute which is called geolink.
```
Comment (ger-benjamin) [inline doc/source/changes.rst]: ***an** attribute

File: doc/source/changes.rst [hunk 5]
```diff
@@ -0,0 +1,555 @@
...
+
+
+.. _changes-new-config-options:
+
+New configuration options in yml
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. _changes-new-config-mapfish-print:
+
+Mapfish print
```
Comment (ger-benjamin) [inline doc/source/changes.rst]: MapFish Print (with major cases)

File: doc/source/changes.rst [hunk 6]
```diff
@@ -0,0 +1,555 @@
...
+See the `pyramid_oereb_standard.yml <https://github.com/camptocamp/pyramid_oereb/blob/v1.1.0/pyramid_oereb/standard/pyramid_oereb.yml.mako#L65>`__
+for the correct style of the configuration.
+
+- improved print templates to fit federal definitions as good as possible
+- improved configuration in the ``pyramid_oere.yml`` to better support requirements of different operators (multilingual)
+    - **template_name**:
+        Defines the name of the mapfish print template which is used to provide static extract.
+    - **headers**:
+        Defines the content type which is sent to mapfish print service by mapfish print proxy.
+        This should be set to = `Content-Type: application/json; charset=UTF-8`
```
Comment (ger-benjamin) [inline doc/source/changes.rst]: = is strange here
(And "This should", or "This must" ?)

File: doc/source/changes.rst [hunk 7]
```diff
@@ -0,0 +1,555 @@
...
+.. _changes-new-config-mapfish-print:
+
+Mapfish print
+"""""""""""""
+
+See the `pyramid_oereb_standard.yml <https://github.com/camptocamp/pyramid_oereb/blob/v1.1.0/pyramid_oereb/standard/pyramid_oereb.yml.mako#L65>`__
+for the correct style of the configuration.
+
+- improved print templates to fit federal definitions as good as possible
+- improved configuration in the ``pyramid_oere.yml`` to better support requirements of different operators (multilingual)
```
Comment (ger-benjamin) [inline doc/source/changes.rst]: pyramid_oere.yml -> pyramid_oere**b**.yml

**Comment by ger-benjamin** [comment]
> Would it be possible for you to add a little doc about the print config

I'll look for that

**Comment by ger-benjamin** [comment]
Big thanks for this doc !

It's a little bit strange to me to see so many lines of python code in the doc. Can't we reference a file instead ?

**Pull request merged**